packages/util-endpoints/src/utils/evaluateTemplate.ts

```diff
@@ -1,38 +1,46 @@
 import { getAttr } from "../lib";
 import { EvaluateOptions } from "../types";
 
-const ATTR_SHORTHAND_REGEX = new RegExp("\\${([\\w]+)#([\\w]+)}", "g");
-
 export const evaluateTemplate = (template: string, options: EvaluateOptions) => {
-  const templateToEvaluate = template
-    // Replace `{value}` with `${value}`
-    .replace(new RegExp(`\{([^{}]+)\}`, "g"), "${$1}")
-    // Replace `{${value}}` with `{value}`
-    .replace(new RegExp(`\{\\$\{([^{}]+)\}\}`, "g"), "{$1}");
+  const evaluatedTemplateArr: Array<String> = [];
```

Comment:
should be `string[]` or `Array<string>`?

Reply:
I was using `string[]` in the past, but `Array<string>` is better as it's more readable for complex arrays.

Like `(string | number | boolean)[]` vs `Array<string | number | boolean>`
We should enforce a standard across our codebase.

I'll use `string[]` as of now, as that's use at other places.